src/index.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+
+// Register means every single file will get processed by babel that we import!
+import 'babel/register'
```

Comment:
You need to activate `experimental` and you can't use ES6 or ES7 in the current file.

Also you might want to revise `#!/usr/bin/env node` and just have nothing.